config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :channels
```

Comment:
This should be below the "get '*path'" line,  otherwise, it will match any routes that look like "/channels" if there are any in your react app